crates/bevy_render/src/options.rs

```diff
@@ -44,10 +44,16 @@ impl Default for WgpuOptions {
             limits
         };
 
+        let power_preference = if cfg!(target_os = "android") || cfg!(target_os = "ios") {
+            PowerPreference::LowPower
```

Comment:
I don't think this should be done unconditionally. For a game this is the wrong choice most of the time.

Reply:
the default should be the same on all targets, leaving to the user to change their `WgpuOptions` if/when they want